CoreAnalyticSchema/src/au/gov/asd/tac/constellation/graph/schema/analytic/attribute/objects/ExpandedCompositeNodeState.java

```diff
@@ -110,9 +110,7 @@ private void contractSingleVertex(GraphWriteMethods wg, final int vxId, final Re
         idToCopiedId.put(vxId, copiedId[0]);
 
         // Add to the expansion store those transactions between this vertex and all vertices already added to the expansion store
-        idToCopiedId.entrySet().stream().forEach((entry) -> {
-            GraphRecordStoreUtilities.copyTransactionsBetweenVertices(wg, constituentNodeStore, vxId, entry.getKey(), copiedId[0], entry.getValue());
-        });
+        idToCopiedId.entrySet().stream().forEach(entry -> GraphRecordStoreUtilities.copyTransactionsBetweenVertices(wg, constituentNodeStore, vxId, entry.getKey(), copiedId[0], entry.getValue()));
```

Comment:
I think i liked the multi-line version here - it's too long to easily read.

Reply:
Done